src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1815,7 +1891,7 @@
     examples:
         - name: Delete a virtual network peering
           text: >
-            az network vnet peering delete --name myVnet1toMyVnet2 --resource-group myResourceGroup --vnet-name myVnet1
+            az network vnet peering delete --name MyVnet1toMyVnet2 --resource-group MyResourceGroup --vnet-name MyVnet1
```

Comment:
@tjprescott I would like us to eventually standardize argument values in some way, but for right now I would like feedback on snake-casing at least parts of these arguments to something like `MyVNET1_to_MyVNET2` to make the name delimiting clearer.

Reply:
I agree that these should be generalized, but I would not advocate snake-casing as a lot of names on the service-side would reject those.  I tend to favor the `-g {myRg}`  style with curly-braces, but I wouldn't have any issue with this.

Reply:
If snake-casing causes problems on the service side, let's go with `MyVNet1ToMyVent2`, to fully camel-case.

Reply:
Text should be changed to match the suggested camelcasing: `MyVNet1ToMyVNet2`. Applies to all similar peering commands where the resource names need to be more readable.